templates/tools/dockerfile/test/sanity/Dockerfile.template

```diff
@@ -48,9 +48,10 @@
   #======================================
   # More sanity test dependencies (bazel)
   RUN apt-get install -y openjdk-8-jdk
-  # TOOD(jtattermusch): pin the bazel version
+  # Revert Bazel to an old commit that supports running ./compile.sh 
+  # without a local protoc dependency
   RUN git clone https://github.com/bazelbuild/bazel.git /bazel
-  RUN cd /bazel && ./compile.sh
+  RUN cd /bazel && git reset --hard 19ece7c663cd99d338306e302e0f7ce023becfe3 && ./compile.sh
```

Comment:
`git checkout <SHA1>` does the same and is a bit less cryptic? Also, I'd prefer to use a tag instead of a sha if possible.